src/nvim/viml/parser/expressions.c

```diff
@@ -2866,7 +2866,7 @@ viml_pexpr_parse_no_paren_closing_error: {}
         case kENodeOperator:
           if (prev_token.type == kExprLexSpacing) {
             // For some reason "function (args)" is a function call, but
-            // "(funcref) (args)" is not. AFAIR this somehow involves
+            // "(funcref) (args)" is not. As far as I remember this somehow involves
```

Comment:
I do not think we should start expanding a bunch of acronyms. Use google, people. Or chatgpt.

Reply:
Noted.